src/sentry/scim/endpoints/utils.py

```diff
@@ -100,26 +100,23 @@ def parse_filter_conditions(raw_filters):
     if raw_filters is None:
         return filters
     conditions = raw_filters.split(",")
```

Comment:
- primary change here is instead of looping, we just grab the first filter and use it. if someone tries to add more than 1 filter, that's not supported, so we raise a ValueError. 
- The second change is instead of returning the filter in a list, we return the value instead.